lib/plugins/aws/rollback.js

```diff
@@ -1,6 +1,4 @@
 'use strict';
```

Comment:
I don't think that we will need changes here, but anyway this refactor doesn't harm :)